test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -3186,13 +3380,31 @@ class MultipleDisposalsTriageControllerSpec
           if (expectReturnToSummaryLink) messageFromMessageKey("returns.return-to-summary-link")
           else ""
         )
+        expectedAdditionalIdKeyValues.map(a => doc.select(a.id).html() should include(a.value))
```

Comment:
`shouldBe` is a stronger test here

Reply:
Done.

Reply:
if you do `.text` instead of `.html` here would you be able to do `shouldBe` instead of `should include`?

Reply:
All done.